tests/PHPUnit/Framework/Fixture.php

```diff
@@ -1038,4 +1042,9 @@ public function destroyEnvironment()
         $this->piwikEnvironment->destroy();
         $this->piwikEnvironment = null;
     }
+
+    private function initFromEnvVars()
+    {
+        $this->persistFixtureData = (bool)getenv(self::PERSIST_FIXTURE_DATA_ENV);
```

Comment:
I'd suggest to change that to `$this->persistFixtureData = $this->persistFixtureData || (bool)getenv(self::PERSIST_FIXTURE_DATA_ENV);` so is stays true if that was from outside already

Reply:
👍 will go w/ this approach, nice catch!